### DISCUSSION TITLE
How do I get admin or user access to penpot?

### DISCUSSION THREAD

**Discussion by bobdobbs** [uxbox/uxbox#2508] [Q&A]
After downloading the docker image and getting it running, I've gone to the frontend and run through the process for creating an account.

After that point I can't access the app. 

There is no documentation on acessing an admin interface.

Using the docker manage script I can change the password of the user that I've created. But login requests from that user yield a login error. Changing the password for that user via the manage script doesn't seem to help.

It'd be great if there was some documentation that indicated how to access the admin interface, or do basic admin stuff.

**Comment by niwinz** [comment]
Penpot does not have admin interface right now.
You can't access because you probaly need to set additional flags. The defaults are secure, but not helpful if you expose penpot under HTTP (not HTTPS) on different domain than localhost.

https://help.penpot.app/technical-guide/configuration/#other-flags

![image](https://user-images.githubusercontent.com/843689/198959752-3966fff2-a418-41c2-af09-d6f41ed415c7.png)

**Reply by zackrspv** [reply]
any update on this?, is about 2 years later, was hoping for an admin area to properly manage users and such :)
